src/vs/workbench/contrib/chat/browser/imageUtils.ts

```diff
@@ -66,7 +66,7 @@ export async function resizeImage(data: Uint8Array | string, mimeType?: string):
 					} else {
 						reject(new Error('Failed to create blob from canvas'));
 					}
-				}, 'image/png');
+				}, mimeType || 'image/png');
```

Comment:
we may want to pick only png or jpg. webp images result in 500 errors in Sonnet https://github.com/microsoft/vscode/issues/251450